src/fastaudio/augment/functional.py

```diff
@@ -197,9 +198,9 @@ def colored_noise(shape, exponent, fmin=0, device=None):
     s_scale = s_scale ** (-exponent / 2.0)
 
     # Calculate theoretical output standard deviation from scaling
-    w = s_scale[1:].clone()
```

Comment:
clone was stripping sr. is deepcopy an ok option here or do I need to figure out why sr is getting stripped?